.pre-commit-config.yaml

```diff
@@ -70,3 +70,11 @@ repos:
           ]
         additional_dependencies:
           - tomli==2.0.1
+  - repo: https://github.com/ikamensh/flynt
```

Comment:
Instead of adding a new plugin, can you enable the check in Ruff? https://docs.astral.sh/ruff/rules/#flynt-fly

Reply:
Yes, I will check it.

Reply:
The implementation of Flynt in ruff is still in progress (https://github.com/astral-sh/ruff/issues/2102), the current rule doesn't detect all the usages of the old styles, and I cannot provide the argument `line-length` to deal with long lines. WDYT?

Reply:
Thanks, I was unaware of that. I ran it locally, and it didn't pick up the violations that are caught here.